constantinCurtis/src/main/java/data/collection/task/classes/Address.java

```diff
@@ -22,6 +22,30 @@ public Address(String country, String city, String street) {
         this.street = street;
     }
 
+    public String getCountry() {
```

Comment:
you are never using this method, still we need it?

Reply:
deleted